lib/data/mock/lost_animal_mock.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:miauche/data/interfaces/abstract_api.dart';
+import 'package:miauche/domain/models/animal_model.dart';
+
+class LostAnimalMock extends AnimalAbstractApi {
+  static List<Map> list = [
+    {
+      "animal": "Cachorro",
+      "anotherPlace": "Perto da área verde",
+      "adress": "Rua André Félix da Silva, 495, Novo Horizonte",
+      "anotherPlaceAdress": "Não",
+      "newsName": "Bob",
```

Comment:
O newsName remete ao nome da notícia, ou seja, seria algo como "Cachorrinho perdido nas redondezas...". Tem que ser algo chamativo. Usa tua criatividade.